### PR TITLE
Fixed whitespace and markdown consistency in reference docs

### PR DESCRIPTION
**Pull request by PalmEmanuel**
## Description

Removed unnecessary whitespace and adjusted formatting of all CLI Reference pages for markdown consistency.

### Summary
:memo::art::wrench:

This pull request improves the documentation of various `rad` commands and resource policies by fixing formatting, casing, indentation, and readability issues. It replaces tabs with spaces, removes empty code blocks, adds blank lines and links, and capitalizes PowerShell in several files under `docs/content/reference`.

> _We're the `rad` crew and we sail the docs_
> _We fix the tabs and spaces and the empty blocks_
> _We heave and haul on the count of three_
> _We make the Markdown tidy as can be_

### Walkthrough
*  Remove empty code blocks from documentation ([link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-cfaf17663b1c6fdd12427eb256879b86ecb1e0e4f80748334518244ef80a69a1L62), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-7892bbbe75360e62e6695d918bcb04f54798fc3baedd5329649b345609473d30L27), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-7892bbbe75360e62e6695d918bcb04f54798fc3baedd5329649b345609473d30L45))
*  Replace tabs with spaces for consistent indentation and Markdown style ([link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-f318d54a7666f7f9d3612806b51b9d11eae11ad74444012a13c16eba217fd41aL32-R38), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-8584c1820f905c665877867e8abd2eea5bf97c38699a579393e4590f9ba7ab44L51-R51), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-db336c6ab4ae46567468cc54bae96d614e72dd9f607bd3b6395cec67eba4a3d7L58-R58), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-3a270c8737d801ddccd6a09bfd9cb42935d4ab885a59dac1fd0209614aa6735aL50-R50), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-1790099e8f40c48ff4d75d498f40903c4be09d7eb04441adc59b24ef0dead56dL54-R54), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-a2884d4add3a38782818168420bcd9c56f2d6db3bb6428c5c3c0c95af414f9a8L54-R54), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-9281c7b1e446a6bd28775ce6a1d5aa7f3463bbbc0df0ea474aae941c6830d103L44-R44), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-d6be0c686f2d6d7fa67d7a83e4943540e2b8b774b70a3791477fcabe900c1047L32-R35), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-27d5809eb7562e5a4e674afc51a83da931f2ade2fb40a093bb2b4d77775d4154L36-R36), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-1860a07b3076ee9084bcebaa718bf9943fda1e401dd90675bdfeda35212d8ba5L36-R36), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-b8a90565eb5c83c012e3d16919920825d6a7b4ed1be180614fc2c6d7f747c998L51-R51), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-9d8868d5ad57f9edaefd6a06b4d5b3c8084042aecc7be1b3fbca4b256366e234L77-R78), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-0199fc3114a8f303f3d67899cdf7f3648e7549a34c7da77a6b5e17716b0f9fceL32-R32), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-d2244aec3a5d989367272e76af24a10c9dae5289352e5c4e8e3092067079fa7eL32-R32), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-6354e0c9e48daaf6024b9f562af6d1d0b34c157bc3c99edf4b4deab073544004L32-R32), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-10293254ff874f9dc590c5429e0a04737d1597c70d07879fdd411cddeac5410aL53-R53), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-9e04aa6c1927324ea5df325731e32c1ae3c83507f8b75e9e81d1d6bdea9c847eL50-R50), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-407750ac01af1bce71961baf46800519758715ad984af5a1ddfa0c9a42a52745L27-R42))
*  Add blank lines and a link to improve readability and formatting of `rad_bicep_publish.md` ([link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-b8a90565eb5c83c012e3d16919920825d6a7b4ed1be180614fc2c6d7f747c998L16-R22), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-b8a90565eb5c83c012e3d16919920825d6a7b4ed1be180614fc2c6d7f747c998L28-R31))
*  Capitalize PowerShell for proper casing in `rad_completion_powershell.md` and `rad_completion.md` ([link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-d2244aec3a5d989367272e76af24a10c9dae5289352e5c4e8e3092067079fa7eL11-R11), [link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-9d8868d5ad57f9edaefd6a06b4d5b3c8084042aecc7be1b3fbca4b256366e234L77-R78))
*  Add blank lines to improve readability and formatting of `rad_completion.md` ([link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-9d8868d5ad57f9edaefd6a06b4d5b3c8084042aecc7be1b3fbca4b256366e234L20-R57))
*  Remove extra spaces at the end of the lines in `rad_credential_register.md` ([link](https://github.com/radius-project/docs/pull/978/files?diff=unified&w=0#diff-9e04aa6c1927324ea5df325731e32c1ae3c83507f8b75e9e81d1d6bdea9c847eL27-R30))

### CODE REVIEW

**Comment by PalmEmanuel** [comment]
I was asked to split https://github.com/radius-project/docs/pull/967 in different more logical pieces. I understand this is still a pretty big PR, but it's mainly whitespace fixes for the CLI reference docs.

File: docs/content/reference/cli/rad.md
Comment (AaronCrawfis) [inline docs/content/reference/cli/rad.md:1]: @PalmEmanuel thank you for spotting the whitespace issues here! Unfortunately all the markdown documents under /docs/content/reference/cli are auto-generated by https://github.com/radius-project/radius/blob/main/cmd/docgen/main.go. Any direct changes to these files will be overwritten by the next docs generation.

Could you remove all the cli docs from this PR? Please feel free to check the docs generation script to see if there's a way to fix the whitespace issue as well. Thanks!

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: docs/content/reference/cli/rad.md
Comment (PalmEmanuel) [reply docs/content/reference/cli/rad.md:1]: I suspected as much, I'll take a look at the script instead and restore the CLI reference files 👍

Comment (PalmEmanuel) [reply docs/content/reference/cli/rad.md:1]: Restored the CLI docs, might dive into the script at a later point. My own blog uses Hugo but I haven't poked too much at Go yet :)

You can merge this one for now (if the rest looks good), and I'll open other PRs for the other changes I have.

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**